src/app/charts/core/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: core
-version: 0.7.5
+version: 0.7.6
 appVersion: "3.0.0"
 description: A Helm chart for core ArkCase and ConfigServer
 type: application
 dependencies:
   - name: common
-    version: ~0.7.0
+    version: ~0.7.9
```

Comment:
My thinking here was that 0.7.0 through 0.7.8 would not work for OIDC deployments.  Since Shinnick is talking about using OIDC for all deployments (via keycloak.armedia.com or the appdev keycloak), then 0.7.0 through 0.7.8 wouldn't work for most deployments of legacy ArkCase; hence the version bump to ~0.7.9.  If this really isn't a problem at all, or if Helm will always take the latest 0.7.x regardless I'll back out this one.

Reply:
I see your point.

What matters to Helm first and foremost is the version of the library bundled within the chart, which will be yours. The second criteria could be to have Helm update dependencies on deployment, which would still land it on yours.

I'm OK with either approach, really. This used to be a headache when we had separate charts for everything, but now that it's a single dependency, we can probably live with this.